themes/default/content/docs/pulumi-cloud/cloud-rest-api/_index.md

```diff
@@ -2759,18 +2760,27 @@ If omitted, results are sorted according to their search relevance. If there is
 
 If specified more than once, the first parameter is the primary sort order and subsequent parameters control additional sorting criteria.
 
+Special consideration needs to be given if using this in tandem with the **cursor** parameter, see below.
+
 Allowed values: created, custom, delete, id, modified, module, name, package, parent.urn, pending, project, protected, provider.urn, stack, type, urn.
 
 **asc**: Whether to return results in ascending or descending sort order.
 
 **size**: How many results to return at a time.
 
+**paginate**: If requesting multiple pages of results, the pagination method to use. Should be "cursor" or "page"
+This tells the API that you would like to receive a continuation token for paging additional results.
+
 **page**: The page of results to return.
-The page parameter can only be used to fetch up 10,000 resources. If a query matches more than 10,000 resources, the "continue" parameter should be used instead.
+The page parameter can only be used to fetch up 10,000 resources. If a query matches more than 10,000 resources, the "cursor" parameter should be used instead.
 
-**continue**: A continuation token for pagination.
+**cursor**: A continuation token for pagination.
 Only available on Enterprise plans.
 
+To receive a cursor from the API, you need to issue a `paginate="cursor"` query parameter with your initial request. This informs the API that it should return a cursor as part of the JSON response.
```

Comment:
How about something like:

```
`paginate=cursor` returns a continuation token that can be used to page through all results of a query, not just the first 10,000. Cursor pagination will ignores any sort parameters in the query and instead returns results sorted by a stable resource ID.
```

Reply:
+1, but with `Cursor pagination ignores`.

Reply:
This is not entirely correct. We do respect any custom sorts applied by the customer--it is the relevance sorting that they see for a query issued in the UI that we drop if paginating.

This decision was made because sorting is _fine_, provided it is deterministic/stable. It was the relevance sort, which can yield different values on every request regardless of underlying data changes, that caused the bug and that we dropped.

All of our other fields are going to be stable unless a stack update finalizes as the pagination requests are in-flight, which I've called out. Happy to reword this.

Reply:
IMO this is too complicated and unnecessary to explain in detail. I would just say something accurate but vague instead:

```
Cursor pagination does not guarantee that results are returned in sorted order.
``